packages/node_modules/node-red/settings.js

```diff
@@ -9,73 +9,30 @@
+ *  - Node Settings
+ *
  **/
 
 module.exports = {
-    // The file containing the flows. If not set, it defaults to flows_<hostname>.json
-    flowFile: 'flows.json',
-
-    // To enabled pretty-printing of the flow within the flow file, set the following
-    //  property to true:
-    //flowFilePretty: true,
-
-    // the tcp port that the Node-RED web server is listening on
-    uiPort: process.env.PORT || 1880,
-
-    // By default, the Node-RED UI accepts connections on all IPv4 interfaces.
-    // To listen on all IPv6 addresses, set uiHost to "::",
-    // The following property can be used to listen on a specific interface. For
-    // example, the following would only allow connections from the local machine.
-    //uiHost: "127.0.0.1",
-
-    // Retry time in milliseconds for MQTT connections
-    mqttReconnectTime: 15000,
-
-    // Retry time in milliseconds for Serial port connections
-    serialReconnectTime: 15000,
-
-    // Retry time in milliseconds for TCP socket connections
-    //socketReconnectTime: 10000,
-
-    // Timeout in milliseconds for TCP server socket connections
-    //  defaults to no timeout
-    //socketTimeout: 120000,
-
-    // Maximum number of messages to wait in queue while attempting to connect to TCP socket
-    //  defaults to 1000
-    //tcpMsgQueueSize: 2000,
-
-    // Timeout in milliseconds for HTTP request connections
-    //  defaults to 120 seconds
-    //httpRequestTimeout: 120000,
-
-    // Maximum buffer size for the exec node
-    //  defaults to 10Mb
-    //execMaxBufferSize: 10000000,
-
-    // The working directory to handle relative file paths from within the File nodes
-    //  defaults to the working directory of the Node-RED process.
-    //fileWorkingDirectory: "",
 
-    // Timeout in milliseconds for inbound WebSocket connections that do not
-    // match any configured node.
-    //  defaults to 5000
-    //inboundWebSocketTimeout: 5000,
+/*******************************************************************************
+ * Flow File and User Directory Settings
+ *  - flowFile
+ *  - credentialSecret
+ *  - flowFilePretty
+ *  - userDir
+ *  - nodesDir
+ ******************************************************************************/
 
-    // The maximum length, in characters, of any message sent to the debug sidebar tab
-    debugMaxLength: 1000,
-
-    // The maximum number of messages nodes will buffer internally as part of their
-    // operation. This applies across a range of nodes that operate on message sequences.
-    //  defaults to no limit. A value of 0 also means no limit is applied.
-    //nodeMessageBufferMaxLength: 0,
-
-    // To disable the option for using local files for storing keys and certificates in the TLS configuration
-    //  node, set this to true
-    //tlsConfigDisableLocalFiles: true,
-
-    // Colourise the console output of the debug node
-    //debugUseColors: true,
+    // The file containing the flows. Defaults to flows_<hostname>.json
```

Comment:
should this now say  defaults to..... if not set below. or something ?

Reply:
It did use to say 'if not set below' - but it felt redundant to say that.

Reply:
Changed to:
```
// The file containing the flows. If not set, it defaults to flows_<hostname>.json
```